torch_xla/csrc/ops/expand_symint.cpp

```diff
@@ -0,0 +1,66 @@
+#include "torch_xla/csrc/ops/expand_symint.h"
+
+#include "absl/strings/str_join.h"
+#include "tensorflow/compiler/xla/client/lib/constants.h"
+#include "tensorflow/compiler/xla/xla_client/debug_macros.h"
+#include "torch/csrc/lazy/core/helpers.h"
+#include "torch/csrc/lazy/core/util.h"
+#include "torch_xla/csrc/data_ops.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input,
+                           const std::vector<int64_t> upper_bounds,
+                           const std::vector<bool> dynamic_dims) {
+  return xla::ShapeUtil::MakeShape(GetXlaShape(input).element_type(),
+                                   {upper_bounds}, {dynamic_dims});
+}
+
+std::vector<torch::lazy::Value> GetValues(
+    const torch::lazy::Value& input,
+    const std::vector<torch::lazy::Value>& dimensions) {
+  std::vector<torch::lazy::Value> values = dimensions;
+  values.insert(values.begin(), input);
```

Comment:
this might be more efficient 

```cpp
std::vector<torch::lazy::Value> values;
values.reserve(dimensions.size() + 1); // to avoid reallocs
values.insert(dimensions.begin(), dimensions.end(), values.end();
```